frontend/BossAlienMediaPlayer/src/app/icecast-stream/icecast-stream.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, ElementRef, OnInit, ViewChild} from '@angular/core';
+import { environment } from 'src/environments/environment';
+
+const volumeKey: string = "streamVolume";
+
+@Component({
+  selector: 'bamp-icecast-stream',
+  templateUrl: './icecast-stream.component.html',
+  styleUrls: ['./icecast-stream.component.scss']
+})
+
+export class IcecastStreamComponent implements OnInit {
+
+  constructor() { }
+
+  @ViewChild("IcecastStreamAudio") audioElement: ElementRef;
+
+  public streamSource: string;
+  public isStreaming: boolean;
+  public currentVolume: number;
+
+  ngOnInit(): void {
+    this.isStreaming = environment.includeAudioStream;
+    if (this.isStreaming == false)
+    {
+      return;
+    }
+    this.streamSource = "http://" + window.location.hostname + ":8000/bamp"
```

Comment:
Could you change this URL to ```this.streamSource = "https://" + window.location.hostname + "/icecast/bamp"```? This should fix problems for all browsers

Reply:
This is addressed in commit 71eb941